### PR TITLE
Start recording Certification data in PostgresDB

### PR DESCRIPTION
**Pull request by shanear**

### CODE REVIEW

**Comment by shanear** [comment]
@orischwartz here's our first postgresDB usage. A good amount of refactoring was involved here. Maybe a good time to use Github's shiny new code review feature? 😄

File: spec/models/form8_spec.rb [hunk 1]
```diff
@@ -169,7 +169,7 @@
 
   context ".from_appeal" do
     before do
-      Timecop.freeze
+      Timecop.freeze(Time.utc(2015, 1, 1, 12, 0, 0))
```
Comment (orischwartz) [inline spec/models/form8_spec.rb]: Déjà vu https://github.com/department-of-veterans-affairs/caseflow-efolder/pull/118/commits/d53e71efd7ed9bd39af42bfe9fe2e11316a29441

Comment (shanear) [inline spec/models/form8_spec.rb]: :stuck_out_tongue_winking_eye:

**Comment by orischwartz** [comment]
I actually don't have much feedback on this. Any way to test that models are backed by the right DB config? Pretty straightforward using `establish_connection` and the naming convention but would be nice to verify we won't try to accidentally write to the wrong DB.

**Comment by shanear** [comment]
@orischwartz The models backed by the local postgres are part of the CI test suite. The VACOLS models are all stubbed in the tests.

If we accidentally try to write a Postgres model to VACOLS, we'll know because the CI tests will blow up (the VACOLS DB won't be connected).

If we accidentally try to write a VACOLS model to Postgres, we won't know unless we test the app with VACOLS connected. But we'll know fast because everything will blow up (the table won't exist in the Postgres DB). 

It'd be nice to have some light E2E smoke testing though, that would definitely catch this early.

**Comment by orischwartz** [comment]
Now needs merge resolution but also deployed in UAT

File: config/database.yml [hunk 2]
```diff
@@ -5,29 +5,39 @@
...
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
+production:
   <<: *default
-  database: db/test.sqlite3
+  url: <%= ENV["POSTGRES_URL"] %>
 
 production:
```
Comment (orischwartz) [inline config/database.yml]: Yikes, this needs to be `production_vacols`...

related, I may have inadvertently created a `SCHEMA_MIGRATIONS` table on VACOLS dev...

Comment (shanear) [inline config/database.yml]: Ack you're right. Sorry about that 😓

Comment (shanear) [inline config/database.yml]: Fixed: https://github.com/department-of-veterans-affairs/caseflow-certification/pull/326/commits/d397b98a5cfe14a7c95c92f134332817713d248d

**Comment by shanear** [comment]
@orischwartz I don't know how to appease your requested change...

**Comment by orischwartz** [comment]
Ha...I can override it I think but for me there's a green dropdown on the files tab for reviewing options.

I'll test this out then mark as approved.

**Comment by orischwartz** [comment]
Actually one small comment--maybe we shouldn't yank the analytics events until we have an alternative dashboard? Or is that queued up and ready to go too?

**Comment by shanear** [comment]
@orischwartz  It's not yet, but it shouldn't be long. Also we still have analytics for pageviews, which is most of what we want anyways.

**Pull request merged**